tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -43,16 +43,16 @@ def task(event, context):  # pylint: disable-msg=unused-argument
     LOGGER.debug("event: {event}", event=event)
     try:
         config = event["config"]
-        exclude_file_types = config.get(CONFIG_EXCLUDE_FILE_TYPES_KEY, None)
+        exclude_file_types = config.get(CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS, None)
```

Comment:
Possibly change to `CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS_KEY` to match naming convention

Reply:
updated